packages/2019-transportation/src/components/TransportationCard/index.js

```diff
@@ -34,19 +32,17 @@ TransportationCard.displayName = "TransportationCard";
 
 TransportationCard.propTypes = {
   init: PropTypes.func,
-  isLoading: PropTypes.bool,
-  data: PropTypes.arrayOf(PropTypes.shape({})),
+  data: resourceShape,
   Layout: PropTypes.node
 };
 
 export default connect(
   state => ({
-    isLoading: isTransportationDataPending(state),
-    data: getTransportationData(state)
+    data: api.selectors.getDisturbanceStops(state) // for 2018 package, replace state with state.transportation2019
```

Comment:
Note to self: check into state input methods

Reply:
@jaronheard, my initial 2 cents: This looks like a fantastic tool to cut down on boilerplate when setting up new cards which makes creating new cards more accessible to devs who may not yet understand the ins and outs of redux. I think needing to customize how the state is input is a small price to pay for that and is something we can potentially abstract